Transfer/Sources/ScheduleTransferSectionData.swift

```diff
@@ -67,14 +67,17 @@ struct ScheduleTransferForeignExchangeData: ScheduleTransferSectionData {
                 let destinationCurrency = foreignExchange.destinationCurrency,
                 let rate = foreignExchange.rate {
                 let sourceAmountFormatted = String(format: "%@ %@",
-                                                   sourceAmount.format(with: sourceCurrency),
+                                                   sourceAmount.format(withActualDecimal: sourceCurrency),
                                                    sourceCurrency)
                 let destinationAmountFormatted = String(format: "%@ %@",
-                                                        destinationAmount.format(with: destinationCurrency),
+                                                        destinationAmount
+                                                            .format(withActualDecimal: destinationCurrency),
                                                         destinationCurrency)
-                let rateFormatted = String(format: "%@ = %@",
-                                           "1".format(with: sourceCurrency),
-                                           rate.format(with: destinationCurrency))
+                let rateFormatted = String(format: "%@ %@ = %@ %@",
+                                           "1".format(withActualDecimal: sourceCurrency),
+                                           sourceCurrency,
+                                           rate.format(withActualDecimal: destinationCurrency),
+                                           destinationCurrency)
 
                 rows.append((title: "transfer_fx_sell_confirmation".localized(), value:sourceAmountFormatted))
                 rows.append((title: "transfer_fx_buy_confirmation".localized(), value: destinationAmountFormatted))
```

Comment:
Also remove extra ()

Reply:
Also remove `currencyFormatter.minimumFractionDigits = 2` from both func format(with currencyCode: String?) -> String and func formatToDouble(with currencyCode: String? = nil) -> Double functions